passport/src/strategy.ts

```diff
@@ -49,18 +49,29 @@ class FakeLocalStrategy extends LocalStrategy {
   // Just inherit the constructor from LocalStrategy.
 }
 
-export function makeStrategy(): Auth0Strategy | FakeLocalStrategy {
+
+export function makeStrategies() {
```

Comment:
I think this function belongs in `auth-handler`: it's emphatically *not* a strategy.